### PR TITLE
Add  .telepaste in telegraph.py to paste text content into telegraph

### PR DESCRIPTION
**Pull request by Jigarvarma2005**

### CODE REVIEW

**Comment by Krishna-Singhal** [comment]
@Jigarvarma2005 , can you add that method in single command

**Comment by Jigarvarma2005** [comment]
> @Jigarvarma2005 , can you add that method in single command

Ok lemme try

**Review by Jigarvarma2005 — COMMENTED** [review]
all fixed

File: userge/plugins/utils/telegraph.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ async def telegraph_(message: Message):
             text = content
             header = "Pasted content by @theuserge"
         t_url = post_to_telegraph(header, text)
-        await message.edit(f"**Your text pasted to [telegraph]({t_url})**",disable_web_page_preview=True)
+        await message.edit(f"**Your text pasted to [telegraph]({t_url})**", disable_web_page_preview=True)
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/telegraph.py:56]: break this line and give indents

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: userge/plugins/utils/telegraph.py [hunk 2]
```diff
@@ -19,24 +19,42 @@
 @userge.on_cmd("telegraph", about={
     'header': "Upload file to Telegra.ph's servers",
     'types': ['.jpg', '.jpeg', '.png', '.gif', '.mp4', '.webp'],
-    'usage': "reply {tr}telegraph to supported media : limit 5MB"})
+    'usage': "reply {tr}telegraph to media or text : limit 5MB for media",
+    'examples': " reply {tr}telegraph to `This is header|This is my content`\n (You can use html code)"})
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/telegraph.py:23]: this line also

File: userge/plugins/utils/telegraph.py [hunk 3]
```diff
@@ -19,24 +19,42 @@
...
                 and replied.document.file_name.endswith(
                     ('.jpg', '.jpeg', '.png', '.gif', '.mp4'))
                 and replied.document.file_size <= _T_LIMIT)):
         await message.err("not supported!")
         return
     await message.edit("`processing...`")
+    if replied.text:
+        content = message.reply_to_message.text
+        if "|" in content:
+            content = content.split("|")
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/telegraph.py:45]: add maxsplit=1 in split()

File: userge/plugins/utils/telegraph.py [hunk 4]
```diff
@@ -19,24 +19,42 @@
...
+            if len(content) == 2:
```
Comment (Krishna-Singhal) [inline userge/plugins/utils/telegraph.py:46]: I think no need of if else here,

just write 

`if '|' in replied.text:`
`....content = text.split('|', maxsplit=1)`
`....header = content[0]`
`....text = content[1]`

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

**Comment by rking32** [comment]
Hi @Jigarvarma2005, thanks for this pr

**Pull request merged**